cli.js

```diff
@@ -44,6 +44,14 @@ Usage: $0 <command> [options]` // eslint-disable-line comma-dangle
   .command('create <function> [options]', 'Create a function from template', (yargs0) => {
     yargs0
       .usage('Usage: $0 create <function> [options]')
+      // TODO: should runtime be positional (required)?
```

Comment:
I don't see a way to have a sane default for runtime. If it's a python user, genering a js function would be a real annoying pain point. Has to be explicit I'm afraid.

Reply:
position vs. required: a good question. I don't like required named options. But on the other hand, I particularly dislike having *multiple* positional parameters, because it tends to be hard to remember the order, and also unless the values are very self-explanatory, it makes it hard to read.
e.g. `ln -s ... ...` is a terrible terrible CLI. I never remember which is which, because the values you pass give no indication which is which.
however `git push origin master` is fine and dandy. Because `origin` is never the name of a branch and if you read this line you'll never confuse the two.
so `bn create node8 hello`... is it readable enough? probably. but a bit verbose, too.
Maybe stuff like the runtime we should ask the user interactively if he does not specify. I think that's the way serverless framework do things, for example. 
So here's a proposal: `bn create RUNTIME FUNC_NAME`.
If a user just passes `bn create hello` then it's easy to detect that `hello` is not a valid runtime and give an *informative* error. In which case we can query the user interactively for a runtime.
It will look like this:

```
$ bn create hello

Creating skeleton for function "hello".
Please choose the runtime:
* python2
* node8
* pypy
```
If the user just made a mistake in the runtime name, eg.. `bn create python hello` we can, again, give an informative error message.